tests/PoolTest.php

```diff
@@ -80,7 +79,7 @@ public function testPrefix()
             ->will($this->returnValue($loggerConfig));
 
         $factory = $this->getMock('\Phlib\Logger\Factory');
-        $logger  = $this->getMock('\Phlib\Logger\Gelf');
+        $logger  = $this->getMockBuilder('\Phlib\Logger\LoggerType\Stream')->disableOriginalConstructor()->getMock();
```

Comment:
Yeah, it was annoying to have to find these manually, when using `::class` would have easily shown up those that were no longer valid (and actually, IDE would have automatically renamed them!)